test/controllers/fhir/r4/api_controller_test.rb

```diff
@@ -720,6 +721,74 @@ def setup_patients
     assert json_response['extension'].filter { |e| e['url'].include? 'isolation' }.first['valueBoolean']
   end
 
+  test 'SYSTEM FLOW: should update Patient via update and set omitted fields to nil ' do
+    # Possible update request that omits all fields that can be updated except for the "active" field.
+    patient_update = {
+      'id' => @patient_2.id,
+      'identifier' => '100000',
```

Comment:
Unless this `identifier` has some meaning I'm missing outside of the `Patient.identifier` in FHIR, I think we may want to update this to not point `identifier` at a single string, since the `Patient.identifier` field in FHIR is an array of objects. See the spec [here](https://www.hl7.org/fhir/patient-definitions.html#Patient.identifier). I don't think we use `identifier` in the `from_fhir` method right now, but if we start, and started validating it, I suspect this could cause some issues. Sorry if we use `identifier` in some other way I'm missing and this is way off base.

Reply:
Glad you pointed this out - I was using a test request body that I had actually taken from a real user which clearly was using that incorrectly. No need to include it here so I removed it from these tests.